### ISSUE TITLE
Shouldn't this be a plus?

### ISSUE THREAD

**Issue by brando90** [markdtw/meta-learning-lstm-pytorch#1]
https://github.com/markdtw/meta-learning-lstm-pytorch/blob/fcc68baad71a3293edeb227a1676de9e9edea9c3/metalearner.py#L67

**Comment by brando90** [comment]
no it's not because the learning rate is output of a sigmoid which is always positive, so to go DOWN the landscape we need negative.
